.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "Chemprop - CPU",
+    "build": {
+        "context": "..",
+        "dockerfile": "../Dockerfile"
+    },
+    "settings": {
+        "terminal.integrated.shell.linux": "/bin/bash"
+    },
+    "postCreateCommand": "pip install -e \".[hpopt,dev,docs,test,notebooks]\"",
```

Comment:
does `.[<...>]` need to be quoted here?

Reply:
I didn't try it here without the quotes, but this is how I normally do it on the command line. Or maybe it's only required on the command line when using `zsh`? I'm not sure.

Reply:
FWIW I use `bash` (which should be the default here) and don't use quotation marks. It's good for safety, but at the point you start escaping characters "safety" is not really applicable.